src/Rules/Generate.hs

```diff
@@ -177,6 +185,31 @@ generateRules = do
   where
     file <~ gen = file %> \out -> generate out emptyTarget gen
 
+-- | Generate scripts the build system requires. For now we generate the
+-- @ghc-split@ script from it's literate perl source.
+generateScripts :: Rules ()
+generateScripts = do
+    -- how to translate literate perl to perl.
+    -- this is a hack :-/
+    "//*.prl" %> \out -> do
+        let src = out -<.> "lprl"
+        path <- builderPath Unlit
+        need [path]
+        unit $ cmd [path] [src] [out]
+
+    -- ghc-split is only a perl script.
+    let ghcSplit = "inplace/lib/ghc-split" -- See system.config
+    let ghcSplitBin = "inplace/lib/bin/ghc-split" -- See ConfigHs.hs
```

Comment:
That got me very confused as well. `cfg/system.config` has `unlit` in `inplace/lib`. Yet we need it in `inplace/lib/bin` as well. Something fishy is going on here :(